src/liquid/components/move_result.html

```diff
@@ -9,26 +9,27 @@
   </ul>
   <ul>
     <li><span class="tag is-success" style="margin: 0 10px;">After</span></li>
+    <li><span style="margin: 0 10px;">{{ fromLineage.first.name }}</span></li>
     {% for lineage in toLineage %}
     {% if lineage %}
       <li><a href="https://{{ session.authInfo.domain }}/#/contacts/{{ lineage.id}}">{{ lineage.name }}</a></li>
     {% endif %}
     {% endfor %}
-    <li><span style="margin: 0 10px;">{{ fromLineage.last.name }}</span></li>
   </ul>
 </nav>
 
 <section class="hero {% if isError %}is-danger{% else %}is-info{% endif %}">
   <div class="hero-body">
     <div>
-      <button
+      <!-- <button
         class="button is-right"
         style="float: right;"
         onclick="navigator.clipboard.writeText(`{{ command }}`);"
       >
         <span class="material-symbols-outlined">content_copy</span>
       </button>
-      {{ command }}
+      {{ command }} -->
+      A worker is taking care of this job, you can please check it <a target="_blank" style="color:chartreuse; font-weight: bold;" href="{{ jobsBoardUrl }}">here</a>.
```

Comment:
what should we actually show here?  one option would be to show nothing and remove this page entirely. then we can just show something like a [toast](https://rfoel.github.io/bulma-toast/)?